lib/server.js

```diff
@@ -130,13 +135,23 @@ module.exports = async config => {
             eventFactory: config.eventFactory,
             collectionFactory: config.collectionFactory,
             buildClusterFactory: config.buildClusterFactory,
-            ecosystem: config.ecosystem
+            ecosystem: config.ecosystem,
+            release: config.release
         };
 
         const bellConfigs = await config.auth.scm.getBellConfiguration();
 
         config.auth.bell = bellConfigs;
 
+        if (config.release && config.release.cookieName) {
+            server.state(config.release.cookieName, {
+                path: '/',
+                ttl: (config.auth.sessionTimeout / 60) * 60 * 1000, // setting to 2 minutes
```

Comment:
Hmm, why not just `config.auth.sessionTimeout * 1000` ?

Reply:
Agree, although kind of changes the understanding of config.auth.sessionTimeout being set to 120 minutes.
https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L39-L40